packages/vm/tests/api/runCall.spec.ts

```diff
@@ -262,3 +262,23 @@ tape('Ensure that IDENTITY precompile copies the memory', async (t) => {
 
   t.end()
 })
+
+tape('Throws on negative call value', async (t) => {
```

Comment:
Since the check is in `message.ts` (which I think is the right place), I think we should also test on `runCode` and `runTx`.

Reply:
Sure, I'll add a test for each of those before EOD.

Reply:
Added for runCode, but wasn't able to add one for runTx since I can't figure out how to build a negative value tx at the moment (left a message in the discord about that)